### PR TITLE
complete profile screen

### PR DESCRIPTION
**Pull request by Ali873-debug**

### CODE REVIEW

**Review by AmeerAmjed — CHANGES_REQUESTED** [review]
implement design screen not completed?

**Review by AmeerAmjed — APPROVED** [review]

File: app/src/main/res/drawable/large_textview.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <solid android:color="@color/card"/>
+    <corners android:radius="28dp"/>
```
Comment (msalarab) [inline app/src/main/res/drawable/large_textview.xml]: hard code

File: app/src/main/res/drawable/thumb.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item android:state_checked="true">
+        <shape android:shape="oval">
+            <solid android:color="@color/white"/>
+            <size android:width="24dp"
```
Comment (msalarab) [inline app/src/main/res/drawable/thumb.xml]: hard code

File: app/src/main/res/drawable/thumb.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+                android:height="24dp"/>
+        </shape>
+    </item>
+    <item android:state_checked="false">
+        <shape android:shape="oval">
+            <solid android:color="@color/white"/>
+            <size android:width="24dp"
```
Comment (msalarab) [inline app/src/main/res/drawable/thumb.xml]: hard code

File: app/src/main/res/drawable/track.xml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+
+
+    <item android:state_checked="true">
+        <shape android:shape="rectangle">
+            <solid android:color="@color/primary" />
+            <size android:height="10dp" />
```
Comment (msalarab) [inline app/src/main/res/drawable/track.xml]: hard code

File: app/src/main/res/drawable/track.xml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+            <corners android:radius="100dp"/>
+        </shape>
+    </item>
+
+    <item android:state_checked="false">
+        <shape android:shape="rectangle">
+            <solid android:color="#DFDFE0" />
```
Comment (msalarab) [inline app/src/main/res/drawable/track.xml]: hard code

**Review by msalarab — CHANGES_REQUESTED** [review]
ps

**Review by mosayed01 — APPROVED** [review]

**Pull request merged**